sse2neon.h

```diff
@@ -5247,6 +5251,71 @@ FORCE_INLINE __m128d _mm_blendv_pd(__m128d _a, __m128d _b, __m128d _mask)
 #endif
 }
 
+
+#if defined(__aarch64__)
+typedef struct {
```

Comment:
It seem that the structure definitions between Aarch64 and ARM32 had some overlapping. Can you rewrite as following?
```c
typedef struct {
    uint16_t res0;
    uint8_t res1 : 6;
    uint8_t bit22 : 1;
    uint8_t bit23 : 1;
    uint8_t res2;
#if defined(__aarch64__)
    uint32_t res3;
#endif
} fpcr_bitfield;

typedef union {
    fpcr_bitfield field;
#if defined(__aarch64__)
    uint64_t value;
#else
    uint32_t value;
#endif
} reg;

```

Reply:
Can you tweak the struct definitions?

Reply:
It was my fault. I was not aware of the updated code.